### ISSUE TITLE
[v4] Navbar toogler ontop of brand when screen size is 575 or below in width

### ISSUE THREAD

**Issue by joacar** [twbs/bootstrap#22606]
- Windows 10
- Chrome 58.0.3029.96 
- Using bootstrap 4 (modified an existing ASP.NET MVC template for Bootstrap 3)

On small screens (phone) the container class in navbar lose its `width: 100%` casuing the navbar toogler to be aligned above the brand. Its `flex-end` is not honored.

Adding the below fixes the issue. 
```css
@media (max-width: 991px)
.navbar-toggleable-md > .container {
    width: 100%;
}
```

**Edit** ~I'll sneak in an unrelated question I can't find the answer to: How do I set a custom checkbox selected on page load? Is this only possible by JavaScript and if so - do I target the input?~
Just cause I couldn't find any class changes I assumed I had to resolve to other means, but just setting `checked` worked. No big surprise - but silly me missed it!

**Update 2:** Sorry for not posting the [JSBin](https://jsbin.com/zekigifavu/edit?html,js,output)!

**Comment by cr101** [comment]
@joacar can you send a reduced test case using JSBin

**Comment by joacar** [comment]
@cr101 Sorry not posting the [JSBin](https://jsbin.com/zekigifavu/edit?html,js,output)!

**Comment by cr101** [comment]
@joacar this is a duplicate of [#22028](https://github.com/twbs/bootstrap/issues/22028) which has been fixed in [#21722](https://github.com/twbs/bootstrap/pull/21722)

**Comment by joacar** [comment]
@cr101 Perfect, thanks! Sorry taking your time when there already was an open issue - I were to specific in my search so didn't found it.
